lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -864,7 +864,7 @@ function dosomething_reportback_get_reportback_total_by_nid($nid, $type = 'quant
     $query->fields('rb', array('quantity'));
     $query->addExpression('SUM(quantity)', 'quantity');
     $result = $query->execute()->fetchAll();
-    $quantity = $result[0]->quantity;
+    $quantity = (int) dosomething_helpers_get_variable('node', $nid, 'sum_rb_quantity_override') + (int) $result[0]->quantity;
```

Comment:
Is campaign theme the only place this is needed? If so I can just add this in there.